packages/scripts/scripts/test-e2e.js

```diff
@@ -12,7 +12,6 @@ process.on( 'unhandledRejection', ( err ) => {
 /**
  * External dependencies
  */
-/* eslint-disable-next-line jest/no-jest-import */
```

Comment:
No longer a valid rule, it was removed in https://github.com/jest-community/eslint-plugin-jest/pull/1220.